sycl/cmake/modules/FetchUnifiedRuntime.cmake

```diff
@@ -116,7 +116,7 @@ if(SYCL_UR_USE_FETCH_CONTENT)
       CACHE PATH "Path to external '${name}' adapter source dir" FORCE)
   endfunction()
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
+  set(UNIFIED_RUNTIME_REPO "https://github.com/zhaomaosu/unified-runtime.git")
```

Comment:
to be updated after UR PR merges.

Reply:
When there are testing redirects like this, I would prefer that unified runtime reviewers don't give a formal approval until it has been addressed. We risk that the gatekeepers miss a comment like this and merge.

Reply:
Ok, will keep that in mind.